### PR TITLE
Adds gb details to stanley reisner ideal

### PR DESCRIPTION
**Pull request by antonydellavecchia**
Adds groebner basis details to ideal

### CODE REVIEW

**Comment by thofma** [comment]
@ederc or @jankoboehm: Is there a way to set a universal Gröbner basis of an ideal (see the comment in the code)?

**Comment by ederc** [comment]
> @ederc or @jankoboehm: Is there a way to set a universal Gröbner basis of an ideal (see the comment in the code)?

No, at the moment we have no such mechanism.

File: src/Combinatorics/SimplicialComplexes.jl [hunk 1]
```diff
@@ -327,8 +327,12 @@ Ideal generated by
 ```
 """
 function stanley_reisner_ideal(R::MPolyRing, K::SimplicialComplex)
-    mnf = minimal_nonfaces(IncidenceMatrix, K)
-    return ideal([ R([1], [Vector{Int}(mnf[i,:])]) for i in 1:Polymake.nrows(mnf) ])
+  mnf = minimal_nonfaces(IncidenceMatrix, K)
+  gens = [ R([1], [Vector{Int}(mnf[i,:])]) for i in 1:Polymake.nrows(mnf) ]
+  # I is actually a universal groebner basis but I am not sure if there is a way
```
Comment (antonydellavecchia) [inline src/Combinatorics/SimplicialComplexes.jl]:
```suggestion
  # currently no way to set as a universal groebner basis
```

File: src/Combinatorics/SimplicialComplexes.jl [hunk 2]
```diff
@@ -327,8 +327,12 @@ Ideal generated by
 ```
 """
 function stanley_reisner_ideal(R::MPolyRing, K::SimplicialComplex)
-    mnf = minimal_nonfaces(IncidenceMatrix, K)
-    return ideal([ R([1], [Vector{Int}(mnf[i,:])]) for i in 1:Polymake.nrows(mnf) ])
+  mnf = minimal_nonfaces(IncidenceMatrix, K)
+  gens = [ R([1], [Vector{Int}(mnf[i,:])]) for i in 1:Polymake.nrows(mnf) ]
+  # currently no way to set as a universal groebner basis
+  # to set that?
```
Comment (antonydellavecchia) [inline src/Combinatorics/SimplicialComplexes.jl]:
```suggestion
```

**Comment by fieker** [comment]
@jankoboehm  will have a look at this one

**Comment by fingolfin** [comment]
So if there is no way to set a universal GB, we can't use/set one in this PR. Maybe someone would like to open an issue requesting this as a feature. In the meantime, let's just merge this for now (it has been open since March 5 with no objections, and is small enough anyway)

**Pull request merged**